system/mount.py

```diff
@@ -358,13 +358,17 @@ def main():
                     res,msg = mount(module, **args)
             elif 'bind' in args.get('opts', []):
                 changed = True
-                cmd = 'mount -l'
+                cmd = 'findmnt -rn %s' % args['name']
```

Comment:
I there find_bin in python?
I could not find such think in exist code.

@jtyr PR rebased.